contracts/access/AccessControlled.sol

```diff
@@ -51,6 +43,17 @@ abstract contract AccessControlled {
         _;
     }
 
+    /// @dev Constructor contract by setting the ACCESS_CONTROLLER and IP_ACCOUNT_REGISTRY addresses.
+    /// @param accessController The address of the AccessController contract.
+    /// @param ipAccountRegistry The address of the IPAccountRegistry contract.
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor(address accessController, address ipAccountRegistry) {
+        if (accessController == address(0)) revert Errors.AccessControlled__ZeroAddress();
+        if (ipAccountRegistry == address(0)) revert Errors.AccessControlled__ZeroAddress();
+        ACCESS_CONTROLLER = IAccessController(accessController);
+        IP_ACCOUNT_REGISTRY = IIPAccountRegistry(ipAccountRegistry);
+    }
+
```

Comment:
Should it be better to call `_disableInitializers()` at the end of the constructor? It would be more secure per suggestion from OZ's doc: https://docs.openzeppelin.com/contracts/4.x/api/proxy#Initializable-_disableInitializers--

----------------
Calling this in the constructor of a contract will prevent that contract from being initialized or reinitialized to any version. It is recommended to use this to lock implementation contracts that are designed to be called through proxies.

Reply:
Good suggestion, but in this case I decided not to add it here because:
- AccessControlled is used both in upgradeable and non upgradeable contracts, since it doesn't have storage of it's own, it's safe
- to call `_disableInitializer()` I would have to make it Initializable, so it won't make sense for the non upgradeable uses.
- it's an abstract contract, so this constructor is not being called directly. It's safe as long as the "last" contract in the constructor chain calls `_disableInitializer()`

I considered if doing 2 versions of this contract would be clearer, or it's basically duplicated code we don't need.
If it's confusing I can add upgradeable and non upgradeable versions in a different PR